style.css

```diff
@@ -0,0 +1,775 @@
+  background: #41479b;
+  width: 7px;
+  height: 7px;
+}
+
+.catalog__round_azure {
+  background: #257da9;
+  width: 9px;
+  height: 9px;
+}
+
+.video {
+  display: flex;
+  justify-content: space-around;
+  background: #000;
+  height: 494px;
+  color: #fff;
+}
+
+.video__play {
+  width: 53.7%;
+  height: 345px;
+  background: url(images/video.png) no-repeat;
+  background-size: cover;
+  border: 8px solid rgba(255, 255, 255, 0.8);
+  text-align: center;
+}
+
+.video__play:before {
+  display: inline-block;
+  height: 100%;
+  vertical-align: middle;
+  content: "";
+}
+
+.video__btn {
+  background: url(images/play.png) no-repeat;
+  background-size: cover;
+  height: 46.8px;
+  width: 46.8px;
+  border: none;
+  border-radius: 50%;
+  padding: 0;
+}
+
+.video__column {
+  width: 22.7%;
+}
+
+.stories__title {
+  margin: 0;
+}
+
+.stories__menu {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.stories__column {
+  text-align: justify;
+  width: 26%;
+}
+
+.stories__foto {
+  width: 100%;
+}
+
+.stories__data {
+  text-align: right;
+  font-weight: bold;
+  opacity: 0.3;
+}
+
+.hashtag__foto {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.hashtag__img {
+  width: 14.3%;
+}
+
+.hashtag__img:nth-of-type(1n+2) {
+  margin-left: 0;
+  padding: 0;
+}
+
+.info {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0;
+  padding: 0;
+  background: #000;
+  color: #fff;
+}
+
+@media (max-width:1070px) {
+  body {
+    padding: 0 4.6%;
+  }
+
+  .navigation {
+    margin-bottom: 34px;
+  }
+
+  .navigation__img {
+    width: 61px;
+    height: 64px;
+    margin-right: 3%;
+    padding-top: 3px;
+  }
+
+  .navigation__banner {
+    padding-top: 26px;
+  }
+
+  .navigation__submenu {
+    margin-top: 3px;
+    margin-right: 0.5%;
+  }
+
+  .navigation__link {
+    margin-right: 7.5%;
+  }
+
+  .navigation__form {
+    padding-top: 8px;
+    margin-right: 2.4%;
+  }
+
+  .navigation__search {
+    width: 8.6%;
+  }
+
+  .catalog__wrapper {
+    height: 253px;
+    padding: 75px 0 0 53px;
+    margin-bottom: 60px;
+  }
+
+  .catalog__title {
+    padding-bottom: 5px;
+  }
+
+  .catalog__arrow {
+    width: 6.3%;
+    padding-left: 10px;
+  }
+
+  .catalog__subtitle {
+    margin-bottom: 27px;
+  }
+
+  .catalog__menu {
+    margin-bottom: 44px;
+  }
+
+  .catalog__item {
+    min-width: 0;
+  }
+
+  .catalog__round_grey,
+  .catalog__round_red,
+  .catalog__round_yellow,
+  .catalog__round_blue {
+    width: 6.33px;
+  }
+
+  .catalog__round_black,
+  .catalog__round_peach,
+  .catalog__round_wine,
+  .catalog__round_azure {
+    width: 8px;
+  }
+
+  .catalog__img {
+    width: 170px;
+    margin-bottom: 32px;
+  }
+
+  .catalog__scale {
+    width: 26%;
+  }
+}
+
+@media (max-width: 991px) {
+  body {
+    padding: 0;
+  }
+
+  .navigation {
+    margin-bottom: 17px;
+  }
+
+  .navigation__logo {
+    display: block;
+    margin: 0;
+    position: absolute;
+    top: 41.5px;
+    left: 5.6%;
+    width: 20.5%;
+}
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project.Axion/pullRequest?prid=5456827)